planning/mission_planner/src/mission_planner_lanelet2/mission_planner_lanelet2.cpp

```diff
@@ -153,47 +153,49 @@ bool MissionPlannerLanelet2::isRoutingGraphReady() const { return is_graph_ready
 void MissionPlannerLanelet2::visualizeRoute(
   const autoware_auto_planning_msgs::msg::HADMapRoute & route) const
 {
-  lanelet::ConstLanelets route_lanelets;
-  lanelet::ConstLanelets end_lanelets;
-  lanelet::ConstLanelets normal_lanelets;
-  lanelet::ConstLanelets goal_lanelets;
-
```

Comment:
@shmpwk 
If you add below you can make 50 line diff to one line if Im not wrong
if(!route_handler_.isHandlerReady()) return;

Reply:
You're right thank you.
https://github.com/autowarefoundation/autoware.universe/pull/1155/commits/eb164dd7b1454c8e669b8ef07aaddad5f6923c30